### ISSUE TITLE
Option to disable take-backs for game

### ISSUE THREAD

**Issue by ezhan94** [rbtying/shengji#68]
Prevents players from exploiting it to test throws. Since you can queue up a play anyways, this feature isn't necessary imo.

**Comment by rbtying** [comment]
All other players are notified via game-chat that a take back has occurred, so I don't think the ability to take back a play makes a meaningful difference here?

The implementation currently assumes that players want to play in good faith. For example, there's nothing stopping you from opening a new tab and entering the room using somebody else's name to see their hand.

**Comment by cai-jack** [comment]
just pushed in the codes for this. This actually made a difference in our play. U can't always assume good faith playing in all cases, as it turned out :)

Also, I discovered a nice feature when takeback is disabled. Everytime when a trick is blocked, we took back the play, replayed the blocked cards. As it turned out, you don't actually have to do that. With takeback disabled, I found out following players can only play against the blocked cards! Save a bit of hustle in playing, the previous trick shows the blocked trick, and makes implementing this feature much simpler (I was thinking that I needed to handle this case after takeback is disabled)
